pkgs/by-name/wi/wifitui/package.nix

```
@@ -0,0 +1,24 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub
+}: buildGoModule rec {
+  pname = "wifitui";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "shazow";
+    repo = "wifitui-nix";
+    tag = "v${version}";
+    hash = "sha256-pB+FmnkDfnOSD4319QP867cV9Bu8VWublEd48iTx27c=";
+  };
+
+  vendorHash = "sha256-SEQPc13cefzT8SyuD3UmNtTDgcrXUGTX54SBrnOHJJw=";
+
+  meta = with lib; {
+    description = "TUI for managing Wi-Fi connections";
+    homepage = "https://github.com/greedoftheendless/wifitui-nix";
```

```suggestion
    homepage = "https://github.com/shazow/wifitui";
```

There's no need to maintain a fork overlay.